functests-performance-profile-creator/1_performance-profile_creator/ppc.go

```diff
@@ -0,0 +1,159 @@
+package __performance_profile_creator
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/ghodss/yaml"
+
+	performancev2 "github.com/openshift-kni/performance-addon-operators/api/v2"
+	"github.com/openshift-kni/performance-addon-operators/cmd/performance-profile-creator/cmd"
+	testutils "github.com/openshift-kni/performance-addon-operators/functests/utils"
+)
+
+const (
+	mustGatherPath       = "../../testdata/must-gather"
+	expectedProfilesPath = "../../testdata/ppc-expected-profiles"
+	ppcPath              = "../../build/_output/bin/performance-profile-creator"
+
+	argsDelimiter     = "_"
+	keyValueDelimiter = ":"
+)
+
+var _ = Describe("[rfe_id:xxxxx][ppc] Performance Profile Creator", func() {
+
+	It("[test_id:xxxxx] performance profile creator regression tests", func() {
+		Expect(ppcPath).To(BeAnExistingFile())
+
+		// directory base name => full path
+		mustGatherDirs := getMustGatherDirs(mustGatherPath)
+		// full profile path => arguments the profile was created with
+		expectedProfiles := getExpectedProfiles(expectedProfilesPath, mustGatherDirs)
+
+		for expectedProfilePath, args := range expectedProfiles {
+			cmdArgs := fmt.Sprintf("--disable-ht=%v --mcp-name=%s --must-gather-dir-path=%s "+
```

Comment:
it seems we never use `cmdArgs`, do we? If so, why not just create the `args` slice straight away? as extra bonus, that would make the format strings easier to follow:
```
cmdArgs = []string{
   fmt.Sprintf("--disable-ht=%v", args.DisableHT),
   fmt.Sprintf("--mcp-name=%s", args.MCPName),
  ...
}
```

Reply:
will, do thanks!

Reply:
done